README.rst

```diff
@@ -748,6 +748,47 @@ TemplateSendMessage - ImageCarouselTemplate
         )
     )
 
+FlexSendMessage
+^^^^^^^^^^^^^^^^
+
+.. code:: python
+
+    flex_message = FlexSendMessage(
+        alt_text='hello',
+        contents=BubbleContainer(
+            direction='ltr',
+            hero=ImageComponent(
+                url='https://example.com/cafe.jpg',
+                size='full',
+                aspect_ratio='20:13',
+                aspect_mode='cover',
+                action=URIAction(uri='http://example.com', label='label')
+            )
+        )
+    )
+
+※ You can pass a **dict** to FlexSendMessage#contents as follows:
```

Comment:
[imo]
It seems that there are many properties which can accept dict value by using `Base#get_or_new_from_json_dict_with_types`.

How about informing it in more general place?

Reply:
I recommend the use of **dict** only for `FlexSendMessage`.

Reply:
OK. Then no problem.